filebeat/input/event.go

```diff
@@ -11,24 +11,37 @@ import (
 
 // Event is sent to the output and must contain all relevant information
 type Event struct {
-	common.EventMetadata
-	ReadTime     time.Time
-	InputType    string
-	DocumentType string
-	Bytes        int
-	Text         *string
-	JSONConfig   *reader.JSONConfig
-	State        file.State
-	Data         common.MapStr // Use in readers to add data to the event
+	EventMeta
```

Comment:
So currently we have EventMeta twice, means two copies of the data? What would be the affect if we not have it in the Event? I think we will need in the future processors to have access to the meta data too, but do we need this in the first step?